addon-test-support/scroll-into-view-mock.js

```diff
@@ -0,0 +1,31 @@
+import sinon from 'sinon';
+import { find } from '@ember/test-helpers';
+export default function setupScrollIntoView(hooks) {
+  hooks.beforeEach(function () {
+    this.scrollIntoViewSandbox = sinon.createSandbox();
+    const scrollIntoViewMock = this.scrollIntoViewSandbox.stub();
+    this.scrollIntoViewSandbox.replace(
+      window.Element.prototype,
+      'scrollIntoView',
+      scrollIntoViewMock
+    );
+    // function will check if the passed selector matches any of the elements
+    this.scrollIntoViewCalledWith = (selector) => {
+      let element;
+      // check if it's a string and get the object
+      if (typeof selector === 'string') {
+        element = find(selector);
+      } else {
+        // element was passed in
+        element = selector;
+      }
+      const elementsScrolledTo = scrollIntoViewMock
+        .getCalls()
+        .map((call) => call.thisValue);
+      return elementsScrolledTo.includes(element);
```

Comment:
@abeadam @elwayman02  What are your thoughts on placing the assert inside the function here, so that 
```
assert.ok(this.scrollIntoViewCalledWith(...));
```
becomes
```
this.assertScrollIntoViewCalled(...));
```
and in the setup function we can set the assert message so the user doesn't have to.

Reply:
I would love to do that, but there's no good way to get access to the `assert` method without making the user pass it to us, at which point we should just let them do it instead.